docs/quickstart.md

```diff
@@ -215,11 +219,11 @@ ALTER TABLE `hoodie_rt` ADD IF NOT EXISTS PARTITION (datestr='2015-03-17') LOCAT
 
 
 
-## Querying The Dataset
+### Querying The Dataset
```

Comment:
can we find another more speciic heading here may be?  Having "Query a Hoodie dataset" > "Querying The Dataset" as heading hierarchy seems unclear to me.

Reply:
done